django/http/response.py

```diff
@@ -406,28 +408,41 @@ def _set_streaming_content(self, value):
         if hasattr(filelike, 'close'):
             self._closable_objects.append(filelike)
         value = iter(lambda: filelike.read(self.block_size), b'')
-        self.set_headers(filelike)
+        self._set_headers(filelike)
```

Comment:
I'm not sure either, but this way it just seems more how it should have been from the beginning... imo. And will it ever be more worth? Because i'm pretty sure it's only the _churn_ will only get bigger and messier with time.

Reply:
How about instead/additionally we remove that method from the docs, so people don't try to call it manually, unless they find it themselves to override it?